chipsec/helper/linux/linuxhelper.py

```diff
@@ -454,6 +453,7 @@ def native_write_pci_reg(self, bus, device, function, offset, value, size=4, dom
                 from chipsec.helper.linux.legacy_pci import LEGACY_PCI
                 pci = LEGACY_PCI()
                 value = pci.write_pci_config(bus, device, function, offset, value)
+                return False
```

Comment:
What was the reason behind returning False here? From what I can tell, most things just ignore a return value.

Reply:
Just to make it explict, I you feel against it, I can just drop the line

Reply:
My vote would be to leave it